pages/windows/get-acl.md

```diff
@@ -0,0 +1,13 @@
+# Get-Acl
+
+> Gets the security descriptor for a resource, such as a file or registry key.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/get-acl>.
```

Comment:
We don't include the i18n in MSFT URLs
```suggestion
> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.security/get-acl>.
```

Reply:
I thought it was a good idea to include it because when you `tldr` in your language it would direct to the documentation for the command on the same language you setup in tldr.

Reply:
> I thought it was a good idea to include it because when you `tldr` in your language it would direct to the documentation for the command on the same language you setup in tldr.

We don't include it because it automatically redirects to your native/translated page based on your language/locale setting. For example, Let's say I prefer English Uk and I have it as my locale setting then the page will be automatically redirected.

Reply:
Oh, true. Ok then... I'll fix that.